tests/integration/pilot/istioctl_test.go

```diff
@@ -94,7 +95,7 @@ spec:
     - destination: 
         host: reviews
 `)
-			istioCtl := istioctl.NewOrFail(ctx, ctx, istioctl.Config{})
+			istioCtl := istioctl.NewOrFail(ctx, ctx, istioctl.Config{Cluster: ctx.Environment().Clusters()[0]})
```

Comment:
You're requiring that clusters be set explicitly now? I wonder if we should just make a utility for this since I suspect we'll need it often. Maybe `ctx.DefaultCluster()` (I'd like to eventually see the `Environment` abstraction go away and just be part of `resource.Context`)? That will also make it easy to identify all the places that still need to be updated later on.

Reply:
It's not a requirement